modules/products/bitbucket/helm_release.tf

```diff
@@ -0,0 +1,64 @@
+# Install Helm chart for Bitbucket Data Center.
+
+resource "helm_release" "bitbucket" {
+  name       = local.product_name
+  namespace  = var.namespace
+  repository = local.helm_chart_repository
+  chart      = local.product_name
+  version    = local.bitbucket_helm_chart_version
+
+  values = [
+    yamlencode({
+      replicaCount = 1,
+      bitbucket = {
+        clustering = {
+          enabled = true
+        }
+        resources = {
+          jvm = {
+            maxHeap = local.bitbucket_software_resources.maxHeap
+            minHeap = local.bitbucket_software_resources.minHeap
+          }
+          container = {
+            requests = {
+              cpu    = local.bitbucket_software_resources.cpu
+              memory = local.bitbucket_software_resources.mem
+            }
+          }
+        }
+      }
+      database = {
+        url    = module.database.rds_jdbc_connection
+        driver = "org.postgresql.Driver"
+        credentials = {
+          secretName = kubernetes_secret.rds_secret.metadata[0].name
+        }
+      }
+      #      volumes = {
+      #        localHome = {
+      #          persistentVolumeClaim = {
+      #            create = true
+      #          }
+      #        }
+      #        sharedHome = {
+      #          customVolume = {
+      #            persistentVolumeClaim = {
+      #              claimName = var.pvc_claim_name
+      #            }
+      #          }
+      #          subPath = local.product_name
+      #        }
+      #      }
```

Comment:
I say let's uncomment this block and also `pvc_claim_name` variable, we will replace it with proper code when we add NFS later. 

Or delete the commented part from the code as it is not making sense to keep them.

Reply:
I agree, if we can keep these commented in with no impact on the deployment lets just keep them

Reply:
Updated.

Reply:
🌮